ApiBundle/Transformer/NodeTransformer.php

```diff
@@ -170,13 +173,12 @@ protected function getPreviewLink($scheme, $alias, $encryptedId)
         }
         $domain = $scheme . '://' . $alias->getDomain();
         $routeName = 'open_orchestra_base_node_preview';
-        $parameters = array('token' => $encryptedId);
+        $parameters = array(
+            'token' => $encryptedId,
+            'aliasId' => $aliasId,
+            'nodeId' => $nodeId
+        );
 
-        if ($alias->getPrefix() != "") {
-            $previewLink['name'] .= '/' . $alias->getPrefix();
-            $routeName = 'open_orchestra_base_node_preview_with_prefix';
-            $parameters['languagePrefix'] = $alias->getPrefix();
-        }
```

Comment:
I am not for this approach of the problem : by removing the language prefix, you're serving the preview on another site that the one required by the contributor. Even more, maybe on a non existent one if all sites are configured with a prefix.
Moreover, we can expect a lot of bugs in the rendering of blocks depending on the url.